streams-designer/README.md

```diff
@@ -0,0 +1,155 @@
+# Simple load from Cloudant to Db2 in Streams Designer (Beta)
```

Comment:
Does this `(Beta)` refer to the `Streams Designer (Beta)` or is the tutorial a beta? I'm assuming the former, in which case perhaps we should monospace or italic `Streams Designer (Beta)` to make it clear - and I think the main README should be updated to match.

Reply:
Should the main README be monospaced? And if so, should all of the links be monospaced?

Reply:
We could monospace them all for consistency, but I really was just trying to distinguish the Beta tag from our tutorial vs the Streams Designer.